src/openrct2/ride/VehicleEntry.h

```diff
@@ -87,44 +87,64 @@ enum : uint32_t
     VEHICLE_SPRITE_FLAG_USE_4_ROTATION_FRAMES = (1 << 15),
 };
 
+enum VehicleSpriteGroup : uint8_t
```

Comment:
This is not strongly typed? What was the reasoning? Too much swapping between?? I wouldn't have thought there was any increment/decrement.